shared/video-sdk/develop/ensure-channel-quality/project-implementation/ios.mdx

```diff
@@ -0,0 +1,242 @@
+<PlatformWrapper platform="ios">
+
+### Implement the user interface
+
+This section guides you through the necessary UI changes in the _Get Started_ project interface to implement call quality features.
+
```

Comment:
Can you add a note here please. 

To implement this functionality for UIKit, retrieve an Agora Engine instance with a call to (I can't remember now, trying to find the API call), then use the code here.

Reply:
done